notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -1055,7 +1055,8 @@
     "os.chdir(data_dir)\n",
     "file_suffix = ['flc']\n",
     "dataURIs = mast_api_psf.make_dataURIs(obs, detector=detector, file_suffix=file_suffix)\n",
-    "filename = mast_api_psf.download_request(dataURIs, filename='mastDownload.tar.gz', download_type='bundle.tar.gz')\n",
+    "# filename = mast_api_psf.download_request(dataURIs, filename='mastDownload.tar.gz', download_type='bundle.tar.gz')\n",
+    "filename = filename = mast_api_psf.download_request_bundle(dataURIs, filename='mastDownload.tar.gz')\n",
```

Comment:
Duplicate variable. Change to the following:
```
filename = mast_api_psf.download_request_bundle(dataURIs, filename='mastDownload.tar.gz')
```

Reply:
Corrected.